src/components/modal/modal-add-edit-content.tsx

```diff
@@ -43,169 +60,158 @@ const ModalAddEditContent = ({
   const [releaseDate, setReleasedate] = useState(
     editableMode ? formData?.release_date : ``
   );
+  return (
+    <Formik
+      initialValues={{
+        genres,
+        title,
+        overview,
+        posterPath,
+        runtime,
+        releaseDate,
+      }}
+      validationSchema={validationSchema}
+      onSubmit={(values) => {
+        onModalSubmit({
+          ...formData,
+          genres: values.genres,
+          title,
+          overview,
+          poster_path: posterPath,
+          runtime,
+          release_date: releaseDate,
+        });
+      }}
+    >
+      {({ submitForm }) => (
```

Comment:
You don't need to pass any `formik` props to your form if you use `formik` components
[https://formik.org/docs/tutorial#leveraging-react-context](url)